package.json

```diff
@@ -1,14 +1,22 @@
 {
   "name": "mmm-soccer",
   "version": "2.2.1",
-  "description": "European Soccer Standings Module for MagicMirror2",
+  "description": "Soccer/football standings module for MagicMirror²",
   "repository": {
     "type": "git",
     "url": "git+https://github.com/fewieden/MMM-soccer.git"
   },
   "keywords": [
-    "MagicMirror",
-    "soccer standings"
+    "MagicMirror²",
+    "soccer standings",
+    "football standings",
+    "bundesliga",
+    "premier league",
+    "serie a",
+    "la liga",
+    "ligue 1",
+    "eredivisie",
+    "primeira liga"
```

Comment:
On [the new module list ](https://kristjanesperanto.github.io/MagicMirror-3rd-Party-Modules/) we are using `sport` like a main category. So it would be nice if you add it as keyword.